.github/workflows/deploy-package.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will build pyafipws package and deploy on pypi. 
+# For more information see: https://github.com/py2exe/py2exe/blob/master/.github/workflows/CI.yml
+
+name: Deploy to PyPi
+
+on:
+  workflow_run:
+    workflows: ["python-package", "windows-installer"]
+    types:
+      - completed
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9, 3.11]
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set Up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Build PyAfipWs Package
+      run: |
+        mkdir ./dist
+        pip install wheel
+        python setup.py bdist_wheel sdist
+    - name: Publish Package
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        user: __token__
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        repository_url: https://test.pypi.org/legacy/
+    - name: Test PyAfipWs Upload
+      run: |
+        pip install -i https://test.pypi.org/simple/ PyAfipWs
+    
```

Comment:
Here, we'll need a proper test using wsaa / wsfev1
It could be a single lines running `python -m pyafipws.wsaa` and `python -m pyafipws.wsfev1 --dummy` so it checks that the package is being built/uploaded/installed properly
Also it should check the installed version, that matches with the one being built
Please create a new ticket / issue to track this, so we can merge this PR

Reply:
This will mean we will need to use some certificates to test it so I'll just add that as well